core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -202,12 +202,6 @@ sealed trait ZIO[-R, +E, +A] extends Serializable with ZIOPlatformSpecific[R, E,
    */
   final def as[B](b: => B): ZIO[R, E, B] = self.flatMap(new ZIO.ConstZIOFn(() => b))
 
-  /**
-   * Maps the error value of this effect to the specified constant value.
-   */
-  final def asError[E1](e1: => E1)(implicit ev: CanFail[E]): ZIO[R, E1, A] =
```

Comment:
Should we deprecate for RC18 and then axe it in 1.0?

Reply:
Good idea! Fixed now.